idaes/models/unit_models/tests/test_pbr.py

```diff
@@ -0,0 +1,298 @@
+#################################################################################
```

Comment:
I assume this is still WIP, but this model looks like it would need a case-specific set of tests with their own properties (due to the assumptions about the units for the reaction rate).